admin/src/main/java/edu/unc/lib/dl/admin/controller/ResultEntryController.java

```diff
@@ -61,6 +62,7 @@ String getResultEntry(@PathVariable("pid") String pid, Model model, HttpServletR
             throw new ResourceNotFoundException("The requested record either does not exist or is not accessible");
         }
 
+        childrenCountService.addChildrenCounts(Collections.singletonList(entryBean), accessGroups);
```

Comment:
Can probably skip this for FileObjects, something like `!entryBean
.getResourceType().equals(searchSettings.resourceTypeFile)`